d2renderers/d2svg/d2svg.go

```diff
@@ -44,6 +44,7 @@ const (
 )
 
 var multipleOffset = geo.NewVector(10, -10)
+var borderOffset = geo.NewVector(5, 5)
```

Comment:
should move this to `d2target`, so it can be used both here and `d2sketch` without import cycle issues